instrumentation/mongo/mongo-3.1/library/src/test/java/io/opentelemetry/instrumentation/mongo/v3_1/MongoDbAttributesGetterTest.java

```diff
@@ -0,0 +1,107 @@
+
+import java.util.Arrays;
+import org.bson.BsonArray;
+import org.bson.BsonDocument;
+import org.bson.BsonInt32;
+import org.bson.BsonString;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class MongoDbAttributesGetterTest {
+
+  @Test
+  @DisplayName("should sanitize statements to json")
+  void shouldSanitizeStatementsToJson() {
+    MongoDbAttributesGetter extractor =
+        new MongoDbAttributesGetter(true, DEFAULT_MAX_NORMALIZED_QUERY_LENGTH);
+
+    assertThat(
+            sanitizeStatementAcrossVersions(extractor, new BsonDocument("cmd", new BsonInt32(1))))
+        .isEqualTo("{\"cmd\": \"?\"}");
+
+    assertThat(
+            sanitizeStatementAcrossVersions(
+                extractor,
+                new BsonDocument("cmd", new BsonInt32(1))
+                    .append("sub", new BsonDocument("a", new BsonInt32(1)))))
+        .isEqualTo("{\"cmd\": \"?\", \"sub\": {\"a\": \"?\"}}");
+
+    assertThat(
+            sanitizeStatementAcrossVersions(
+                extractor,
+                new BsonDocument("cmd", new BsonInt32(1))
+                    .append("sub", new BsonArray(singletonList(new BsonInt32(1))))))
+        .isEqualTo("{\"cmd\": \"?\", \"sub\": [\"?\"]}");
+  }
+
+  @Test
+  @DisplayName("should only preserve string value if it is the value of the first top-level key")
+  void shouldOnlyPreserveStringValueIfItIsTheValueOfTheFirstTopLevelKey() {
+    MongoDbAttributesGetter extractor =
+        new MongoDbAttributesGetter(true, DEFAULT_MAX_NORMALIZED_QUERY_LENGTH);
+
+    assertThat(
+            sanitizeStatementAcrossVersions(
+                extractor,
+                new BsonDocument("cmd", new BsonString("c"))
+                    .append("f", new BsonString("c"))
+                    .append("sub", new BsonString("c"))))
+        .isEqualTo("{\"cmd\": \"c\", \"f\": \"?\", \"sub\": \"?\"}");
+  }
+
+  @Test
+  @DisplayName("should truncate simple command")
+  void shouldTruncateSimpleCommand() {
+    MongoDbAttributesGetter extractor = new MongoDbAttributesGetter(true, 20);
+
+    String normalized =
+        sanitizeStatementAcrossVersions(
+            extractor,
+            new BsonDocument("cmd", new BsonString("c"))
+                .append("f1", new BsonString("c1"))
+                .append("f2", new BsonString("c2")));
+
+    // This can vary because of different whitespace for different MongoDB versions
+    assertThat(normalized).containsAnyOf("{\"cmd\": \"c\", \"f1\": \"", "{\"cmd\": \"c\", \"f1\" ");
+  }
+
+  @Test
+  @DisplayName("should truncate array")
+  void shouldTruncateArray() {
+    MongoDbAttributesGetter extractor = new MongoDbAttributesGetter(true, 27);
+
+    String normalized =
+        sanitizeStatementAcrossVersions(
+            extractor,
+            new BsonDocument("cmd", new BsonString("c"))
+                .append(
+                    "f1", new BsonArray(Arrays.asList(new BsonString("c1"), new BsonString("c2"))))
```

Comment:
We'd use static import here, for detail you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports
Please modify all same problems together.